testing/apps/details/Dockerfile.sidecar

```diff
@@ -12,11 +12,11 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM amalgam8/a8-sidecar:alpine
+FROM amalgam8/a8-sidecar:latest
```

Comment:
I backed these changes out and tested again and these are not needed, I'll revert them in the next commit.